server/execution/api/lobby.py

```diff
@@ -54,14 +55,17 @@ def set_name():
     try:
         form = request.get_json()
         name = form["name"]
+        force_update = form["force_update"] == "True" if "force_update" in form.keys() else False
```

Comment:
Why str in one case and bool in other?

Reply:
`form["force_update"] == "True"` results to a boolean. The bool constructor does not parse a bool string to a boolean value. Therefore an extra comparison is required.